app/controllers/membership_applications_controller.rb

```diff
@@ -55,8 +55,7 @@ def create
         create_error(t('.error'))
       end
 
-
-      MembershipApplicationMailer.acknowledge_received(@membership_application).deliver
+      send_new_membership_application_notice_to_admins(@membership_application)
 
```

Comment:
@weedySeaDragon oh I'm sorry. I thought `MembershipApplicationMailer.acknowledge_received(@membership_application).deliver` was also my code that I wrote in the past that did not apply anymore. I did not notice it was someone else's. I will put it back. Thank you for spotting it.

Reply:
@weedySeaDragon "The RSpec specification "email us with questions shows membership email address from ENV['SHF_MEMBERSHIP_EMAIL']"
is failing because the email doesn't have that line in it. (it should)"

Looking at the email template for the `AdminMailer`, the line for specifying the `ENV['SHF_MEMBERSHIP_EMAIL']` is not appropriate since we are emailing admins and not members. So there is no need to add that line to the email template?

Would that also mean we can't use the shared emailed test for the `AdminMailer` spec since the shared examples include adding that line with the `ENV['SHF_MEMBERSHIP_EMAIL`]?

Reply:
Let's leave that line in the email for right now.  We'll review that with the client.  (My thought is that people that are new SHF admins may have questions and having that info in the email for them could be handy.  But the client can give us feedback on that idea and let us know.)

If we need to take it out, then yes, we'll need to change the test for the emails:  we may want to refactor the `shared_example` test.  But let's wait and get feedback from the client first.

Reply:
still needed